src/core/CausalTree.ts

```diff
@@ -236,4 +241,12 @@ export default class CausalTree {
 
     return elements;
   }
+
+  /**
+   * Clears the tree weave and yarns.
+   */
+  clean(): void {
```

Comment:
Is 'clean' an idiomatic name for removing all elements of a data structure? At least in Python and Java it's 'clear' for lists and dicts/maps

Reply:
There are no such methods on Javascript standart libraries (e.g., Array, String and Object), and I don't remember now if there are for other libs. So I just chose the name that seemed best to me.